security/pkg/pki/ca/ca.go

```diff
@@ -128,14 +134,17 @@ func NewSelfSignedIstioCA(caCertTTL, certTTL, maxCertTTL time.Duration, org stri
 		opts.RootCertBytes = caSecret.Data[cACertID]
 	}
 
-	return NewIstioCA(opts)
+	return opts, nil
 }
 
 // NewIstioCA returns a new IstioCA instance.
 func NewIstioCA(opts *IstioCAOptions) (*IstioCA, error) {
 	ca := &IstioCA{
 		certTTL:    opts.CertTTL,
 		maxCertTTL: opts.MaxCertTTL,
+
+		livenessProbe:  probe.NewProbe(),
+		readinessProbe: probe.NewProbe(),
```

Comment:
Not sure why both liveness and readiness exist here -- if they deliver the same signal, you can skip readiness for now.

Reply:
Removed liveness probe

Reply:
Reverted liveness and removed readiness